examples/demo.css

```diff
@@ -132,8 +132,8 @@ h1 {
 
 #gif_element {
   display: block;
-  padding: 50px;
   margin: 0 auto;
-
+  width: 100%;
```

Comment:
Hi, this is great, but actually can we remove the `height` style so that the image is displayed at the correct aspect ratio? Thank you!!! 👍

Reply:
@jywarren i'm afraid this is not css fault. The gif gets generated with static width and height. See [gifshot.createGIF options](https://github.com/publiclab/image-sequencer/pull/442/files/c53c730c9cf263aa78b96e7fcb504a2d5fc1e8f1#diff-5b0e3367cfabe5a0173e80753b827d6cR134)